src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.Linker.targets

```diff
@@ -10,6 +10,7 @@ This file contains the .NET 5-specific targets to customize ILLink
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <Target Name="_PrepareLinking"
+      Condition=" '$(PublishTrimmed)' == 'true' "
```

Comment:
Am I to infer `$(PublishTrimmed)` is set when linking is enabled within `dotnet publish`?

What happens when linking *isn't* enabled during `dotnet publish`?  We still need to run a *subset* of linker steps, even in "Debug config builds", which presumably would be "`dotnet publish` when `$(PublishTrimmed)`=False" (?).  See also f96fcf93e157472072576bcc0a8698302899e8cf.

Reply:
`$(PublishTrimmed)` is the flag that decides if the `ILLink` target should run or not:

https://github.com/dotnet/sdk/blob/d30e9ba1fe915d99cd3455bda9b41cde42775fbe/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets#L32-L35

Our targets will enable this if `$(AndroidLinkMode)` is not `None`.

We could remove this `Condition` and it would not break anything. I just changed this for consistency: we don't really need to change `ILLink` settings if it isn't going to run.